app/javascript/local/dropzone-cfg.js

```diff
@@ -2,6 +2,29 @@
 
 import Dropzone from "dropzone"
 
```

Comment:
I have some concerns that we're now replicating these constants here and in the file_upload_validator. Is there a way that we can have them only in one place?

Reply:
Yeah it's not ideal, i'll explore

Reply:
Note: We considered accessing the rails constants by setting a data attribute in the html and then retrieving and using that in the JS. More information [here](https://github.com/rails/webpacker/issues/1684). However, this approach is  recommended against and so we have decided to leave as is